HALO/wales/main.yaml

```
@@ -0,0 +1,35 @@
+sources:
+  cloudparameter:
+    args:
+      path: "{{CATALOG_DIR}}/wales_cloudtop.yaml"
+    description: 'WALES Lidar cloud top heights'
```

This description doesn't seem to fit the name of the source.
```suggestion
    description: 'WALES-Lidar cloud top height, cloud optical thickness and cloud flag data.'
```
This is indicated in the `wales_cloudtop.yaml` file.